app/endpoints/advert.py

```diff
@@ -282,7 +283,7 @@ async def create_advert(
         )
     except Exception as error:
         hyperion_error_logger.error(
-            f"Error while sending cinema recap notification, {error}"
+            f"Error while sending AMAP recap notification, {error}"
```

Comment:
```suggestion
            f"Error while sending advert notification, {error}"
```

Reply:
I added this try/except, but I'm not sure whether it is really necessary, could this code really fail?

Reply:
For now I think this try except can be useful to monitor if all is going well with notifications